avocado/core/settings.py

```diff
@@ -107,6 +107,8 @@ def convert_value_type(value, value_type):
             value_type = float
         elif value_type == 'list':
             value_type = list
+        elif value_type == 'path':
+            value_type = os.path.expanduser
```

Comment:
I think the rename to `value_type` can be done as the name is *also* appropriate IMHO, but it doesn't add that much value *or* should be mixed with this specific change.

Also, can you please point to where `os.path.expanduser` raises `AttributeError`? I have checked both Python (2.6.6) docs, and the current (3.x) source code and I don't see that.